pulp_2to3_migration/app/plugin/iso/repository.py

```diff
@@ -69,6 +69,8 @@ class IsoDistributor(Pulp2to3Distributor):
         base_config = cls.parse_base_config(pulp2distributor, pulp2_config)
         base_config['base_path'] = pulp2_config.get(
             'relative_url', pulp2distributor.pulp2_repository.pulp2_repo_id)
+        # This fails because of course there is no pulp2_repository set on lone distributors.
+        # So what should be used instead?
```

Comment:
@goosemania This PR should be almost done, but I'm not sure how we want to set up the ```base_path``` in this instance.

Reply:
@dralley ,
s/`pulp2distributor.pulp2_repository.pulp2_repo_id`/`pulp2distributor.pulp2_repo_id`/

There is pulp2_repo_id stored on the distributor model. It was added with the recent PR, same issue was for the importers.

Reply:
Cool, that was the last issue. It seems to work now